src/support/thread_group.c

```diff
@@ -184,6 +184,11 @@ __thread_group_resize(WT_SESSION_IMPL *session, WT_THREAD_GROUP *group, uint32_t
           __wt_open_internal_session(conn, group->name, false, session_flags, &thread->session));
         if (LF_ISSET(WT_THREAD_PANIC_FAIL))
             F_SET(thread, WT_THREAD_PANIC_FAIL);
+        /* If asked, set the default isolation level to ISO_READ_COMMITTED.*/
```

Comment:
Given the hoops you need to jump through to get the default isolation level setup explicitly, I don't think it's worth it.

The behvior is the same without this change, I think we should skip explicitly setting read-committed for this thread, and instead add a comment describing that it's necessary for the thread to operation at read-committed isolation level when it's created.

Reply:
I'll add an assert to make sure that eviction threads are running at read-committed isolation.